File: 2025/python/festa.py
escola = int(input())
mercado = int(input())
loja = int(input())

distancia = 0
if abs(mercado - escola) < abs(loja - escola):
    distancia = abs(mercado - escola) + abs(loja - mercado) + abs(escola - loja)
    
else:
    distancia = abs(loja - escola) + abs(mercado - loja) + abs(escola - mercado)

print(distancia)

File: 2025/C/festa.c
#include <stdio.h>

int main(void) {
    int escola, mercado, loja;
    scanf("%d\n%d\n%d", &escola, &mercado, &loja);

    int rua[3] = {escola, mercado, loja};
    int menor = escola, maior = escola;
    for (char predio = 0; predio < 3; predio++) {
        menor = (rua[predio] < menor) ? rua[predio] : menor;
        maior = (rua[predio] > maior) ? rua[predio] : maior;
    }

    printf("%d\n", (maior - menor) * 2);
}